src/ol/layer/Graticule.js

```diff
@@ -86,9 +86,14 @@ const INTERVALS = [
  * appropriate for conformal projections like Spherical Mercator. If you
  * increase the value, more lines will be drawn and the drawing performance will
  * decrease.
- * @property {Stroke} [strokeStyle='rgba(0,0,0,0.2)'] The
+ * @property {Stroke} [strokeStyle] The
  * stroke style to use for drawing the graticule. If not provided, a not fully
- * opaque black will be used.
+ * opaque black will be used. If not provided, the following stroke will be used:
```

Comment:
This now reads "If not provided, a not fully opaque black will be used. If not provided, the following stroke will be used."

I think you probably meant to change it to "If not provided, the following stroke will be used."

Reply:
Ah, thanks @tschaub. You're absolutely right.